### ISSUE TITLE
Format chooser ignored after adding Image to the queue

### ISSUE THREAD

**Issue by will-moore** [glencoesoftware/NGFF-Converter#26]
I wanted to convert a file to OME-TIFF, so I added the file to the queue via drag and drop, chose the format `OME-TIFF` and ran the conversion.
I only noticed about an hour later that it was converting to NGFF instead of TIFF as the format choice was ignored.

**Comment by will-moore** [comment]
If the choice of format is made at the time a file is added to the queue (and can't be changed afterwards) then maybe there should be a pop-up dialog to ask you to choose each time a file is added (similar to OMERO import dialog).

But it's probably preferable to be able to change the format of any items that are in the queue (and not yet started conversion).

**Comment by DavidStirling** [comment]
The GUI is set up so that jobs settings are determined at the time when you drop them in. This was intended to allow you to mix different conversion settings in the same job list. Individual job output names/formats can be edited by double-clicking on the output file section of the job. At some point we'd planned to have a proper 'edit job' dialog which would allow custom parameters to be given to a single job, it's just not been implemented in the current version.

I'd agree that the current setup can be a bit confusing. I imagine a decent solution might be to add an 'apply these settings to all jobs' button, perhaps below the current 'Run' button. Does that sound reasonable?

**Comment by will-moore** [comment]
I imagine that if I wanted to convert 2 files to NGFF and then 2 more to TIFF, I would add the first 2 to the queue, select `NGFF` and hit run. Then add another 2 to the queue, select `TIFF` (only the last 2 that are not running would be changed to TIFF) and hit run.

I didn't realise I could double-click on a file to edit the output settings. Maybe because the cursor doesn't hint that it's clickable/selectable. 
I could imagine that if I simply selected a file (not double-clicked), then chose a different `Convert to` option on the left of the UI, that would update the selected file? And if you allow users to select multiple files then you wouldn't need an "apply to all" button, which is a bit clunky.

If adding a file (or files) to the queue would automatically select it/them, then choosing `Convert to` -> OME-TIFF would update the selected files, then that would also be quite intuitive.

So, the `Convert to` option would behave like a color-button when creating/editing ROIs... When you create an ROI, it gets assigned the current value, but if you select an existing item, you can change it's value with the same chooser.
The same behaviour could also be applied to the `Output Folder` too.

**Comment by mgheirat** [comment]
I had similar experience with the output directory being ignored after files are added to the queue (but before starting the conversion). I also noticed after the conversion was complete that the output directory is ignored. 

After chatting with @DavidStirling, he said the current functionality will allow that you add files to the queue that go to different output directories as you add them in (once you set the output directory before dropping the files), which is nice. But I would not guess such functionality by default with the current GUI setup, so I think some form of helper or change in GUI to signal that functionality would be very nice to have.
